accumulo/src/main/java/org/vertexium/accumulo/AccumuloGraph.java

```diff
@@ -57,6 +58,7 @@
 import static org.vertexium.util.IterableUtils.singleOrDefault;
 import static org.vertexium.util.IterableUtils.toList;
 import static org.vertexium.util.Preconditions.checkNotNull;
+import static org.vertexium.util.StreamUtils.stream;
 
 public class AccumuloGraph extends GraphBaseWithSearchIndex implements Traceable {
```

Comment:
I'm not sure what to do with `getHistoricalPropertyValues`. I'm leaning toward not including extended data in that list until we have a need for it. Thoughts @mwizeman ?